src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -198,7 +190,7 @@ function WorkspaceAccountingPage({policy}: WithPolicyProps) {
 
     return (
         <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
-            <PaidPolicyAccessOrNotFoundComponent
+            <FeatureEnabledAccessOrNotFoundWrapper
```

Comment:
I think there was a misunderstanding 😅 We should use both `PaidPolicyAccessOrNotFoundComponent` and `FeatureEnabledAccessOrNotFoundWrapper`

Reply:
Ayye! got you, was even about asking. thanks for the clarity.

Reply:
Done ✅